### PR TITLE
Markdown Cheat sheet: Added a new key and corrected some wrong keys

### PR DESCRIPTION
**Pull request by mayank7**
Markdown Cheat sheet: Added a new key and corrected some wrong keys

IA Page: https://duck.co/ia/view/markdown_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Markdown Cheat Sheet](https://duck.co/ia/view/markdown_cheat_sheet)

**Description**: Markdown cheat sheet

**Example Query**: [Markdown Cheat Sheet](https://beta.duckduckgo.com/?q=Markdown Cheat Sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/markdown_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/markdown_cheat_sheet)*

File: share/goodie/cheat_sheets/json/markdown.json [hunk 1]
```diff
@@ -87,11 +91,11 @@
     ],
     "Links": [
       {
-        "key": "\\[link text\\](https://duckduckgo.com)",
+        "key": "\\[text link\\](https://duckduckgo.com)",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/markdown.json]: hmm this seems at odds with: http://daringfireball.net/projects/markdown/syntax

Comment (mayank7) [inline share/goodie/cheat_sheets/json/markdown.json]: @mintsoft I found it in the sourceUrl only:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Comment (moollaza) [inline share/goodie/cheat_sheets/json/markdown.json]: Seeing as we're sending them to the GitHub docs, I think this fix is acceptable.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**